datahub-web-react/src/app/entity/shared/tabs/Dataset/Stats/snapshot/TableStats.tsx

```diff
@@ -31,15 +32,14 @@ const StatContainer = styled.div<{ justifyContent }>`
 export default function TableStats({ rowCount, columnCount, queryCount, users, lastUpdated, lastUpdatedUTC }: Props) {
     // If there are less than 4 items, simply stack the stat views.
     const justifyContent = !queryCount && !users ? 'default' : 'space-between';
-
     return (
         <StatSection>
             <Typography.Title level={5}>Table Stats</Typography.Title>
             <StatContainer justifyContent={justifyContent}>
                 {rowCount && (
                     <InfoItem title="Rows">
                         <Typography.Text strong style={{ fontSize: 24 }}>
-                            {rowCount}
+                            {countFormatter(rowCount)}
                         </Typography.Text>
```

Comment:
for this, lets also make it so hovering over the formatted count provides the full number

Reply:
you can use https://ant.design/components/tooltip/

Reply:
Sure, I have also added the comma separated number for long numbers to view better in tooltip. Specifically showed tooltip over right, as top tooltip is hiding the title and we got enough space in right. 

Updated screen shots; 
![image](https://user-images.githubusercontent.com/29713365/149750893-99b55cf3-e10d-49d2-918e-cfb98a94437b.png)